plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergConnectorTest.java

```diff
@@ -3753,7 +3755,7 @@ private void withTransaction(Consumer<Session> consumer)
                 .execute(getSession(), consumer);
     }
 
-    private void dropTable(String table)
+    protected void dropTable(String table)
```

Comment:
This method shouldn't be used as often as it is.
-- we do not need to assert that DROP TABLE actually works in every single test.
Tests should just do `assertUpdate("DROP TABLE " + table);`

I am not asking you to change existing tests, just keep the method private